translations/ui-orders/en.json

```diff
@@ -202,7 +202,7 @@
   "errors.orderAcqUnitsNotFound": "Acquisitions units assigned to order cannot be found",
   "errors.orderApprovalRequired": "Approval is required to open order",
   "errors.orderClosed": "Order cannot be modified in Closed status",
-  "errors.orderGenericError1": "Order could not be saved.",
+  "errors.orderGenericError": "Order could not be saved.",
```

Comment:
Does the error code from BE changed as well?

Reply:
You are right. I thought it was a typo.